### ISSUE TITLE
Error: Cannot find module `fs-extra`

### ISSUE THREAD

**Issue by ruheni** [omar-dulaimi/prisma-schema-sorter#1]
- **I'm submitting a ...**
  - [x] bug report

- **Summary**
Running `npx prisma-schema-sorter sort --schema="./prisma/schema.prisma"` returns the following error:
```
Cannot find module 'fs-extra'
Require stack:
- /Users/USER/.npm/_npx/5789/lib/node_modules/prisma-schema-sorter/build/main/lib/index.js
- /Users/USER/.npm/_npx/5789/lib/node_modules/prisma-schema-sorter/build/main/bin/cli.js
```

I'm running on node v14.17.1 and npm v6.14.3 and MacOS v12

**Comment by omar-dulaimi** [comment]
Hello @ruheni Glad to see you here.
Thank you for bug report. If you found the issue feel free to submit a PR. Otherwise let me know so I could work on it tonight hopefully.

Thanks again, and happy to see someone from the Prisma team here.

**Comment by omar-dulaimi** [comment]
@ruheni Should be fixed with v1.3.4

Let me know if you face any issues.

**Comment by ruheni** [comment]
Thank you. This was resolved
